addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -877,13 +877,15 @@ const Wysiwyg = Widget.extend({
                         const $colorpicker = $dropdown.find('.colorpicker');
                         const colorpickerHeight = $colorpicker.outerHeight();
                         const toolbarPos = this.toolbar.$el.offset();
+                        const customizePanelOffset = $dropdown.closest('.o_we_customize_panel').offset();
+                        const optionsPanelTop = customizePanelOffset ? customizePanelOffset.top : 0;
```

Comment:
@bso-odoo This works great but I was still wondering why it was needed to handle the case where there is no offset even in unit tests. In fact, your code here should not assume a o_we_customize_panel stuff may even exist: if I am not mistaken, this also handle the opening of colorpicker in backend HTML field (or at least I just tested, and it act the same there... hopefully because of your code and not some other duplicated one?).

Maybe just using `closestScrollable` to find the customize panel would do the trick? And that would be generic to any editor location this code is handling. And don't name the variables using the term "panel" then, name them with something like "toolbar container" which is more generic too.

Reply:
Also applied same change in https://github.com/odoo/odoo/pull/76209 (saas-14.5)